src/main/java/core/basesyntax/supplier/FigureSupplier.java

```diff
@@ -0,0 +1,68 @@
+package core.basesyntax.supplier;
+
+import core.basesyntax.model.Circle;
+import core.basesyntax.model.Figure;
+import core.basesyntax.model.IsoscelesTrapezoid;
+import core.basesyntax.model.Rectangle;
+import core.basesyntax.model.RightTriangle;
+import core.basesyntax.model.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    static final int COLOR_NUMBER = 5;
+    static final int RANDOM_NUMBER = 100;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int figureIndex = new Random().nextInt(COLOR_NUMBER);
```

Comment:
```suggestion
        int figureIndex = random.nextInt(COLOR_NUMBER);
```
lets make Random global, this way you will create just 1 instance of random instead of a new instance each time this method is invoked